docs/central-upgrade.rst

```diff
@@ -76,14 +77,170 @@ You'll be asked to confirm the removal of all dangling images. Agree by typing t
 
 6. Restart the server
 
-  .. code-block:: console
+.. code-block:: bash
+
+  docker-compose up -d
+
+.. _version-specific-instructions:
+
+Version-specific upgrade instructions
+--------------------------------------
+
+.. _central-upgrade-2023.2:
+
+Upgrading to Central v2023.2
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. warning::
+  This upgrade may take more time and disk space than previous updates, as it includes upgrading the PostgreSQL database version.
+
+Run this command to determine whether the server you are upgrading uses the default database or a custom database server:
+
+.. code-block:: bash
+
+  head files/service/config.json.template
+
+If you see ``"host": "postgres"``, you are using the default database. If ``host`` is set to anything else, you are using a custom database server.
+
+.. tabs::
+
+  .. tab:: Default database
+
+    .. warning::
+      Before starting:
+
+      #. :doc:`Back up your server <central-backup>`.
+      #. Make sure you have some time available in case something goes wrong (we recommend at least 2 hours). You may want to announce a maintenance window.
+      #. Review upgrade notes for all versions between your current version and the version you are upgrading to.
```

Comment:
In a step below, it says "Log into the web interface and do some quick spot checks. For example, verify that submission counts and latest submission dates look right and try a data export." Would have liked something to tell me to make a note of these numbers before hand.